### PR TITLE
Feature/count points

### PR DESCRIPTION
**Pull request by eliasreis54**

### CODE REVIEW

File: lib/points.dart [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import 'package:flutter/material.dart';
+
+class PointsRender {
+  void render(Canvas canvas, int points, Size screenSize) {
+    final width = ((screenSize.width / 2) - 30);
+    final position = Offset(width, 100);
+
+    final text = TextPainter(
```
Comment (erickzanardo) [inline lib/points.dart]: Flame already has a nice class to handle this, it is called TextConfig https://flame-engine.org/docs/text.md

File: lib/restart_game.dart [hunk 2]
```diff
@@ -8,24 +8,24 @@ class RestartGame extends StatelessWidget {
...
-                      textDirection: TextDirection.rtl,
-                  ),
-                  RaisedButton(
-                      onPressed: this.onPressRestart,
-                      child: Text(
-                          'Reiniciar',
-                      )
-                  ),
-                ],
+      textDirection: TextDirection.rtl,
```
Comment (erickzanardo) [inline lib/restart_game.dart:11]: For the game UI, Flame has a nice api called widgets overlay, which can make this logic of your way easier, I can explain it later if you want

File: lib/points.dart (hunk 1 above)
Comment (eliasreis54) [reply lib/points.dart]: Thanks! It's more simple than i was doing

File: lib/restart_game.dart (hunk 2 above)
Comment (eliasreis54) [reply lib/restart_game.dart:11]: Yes! I want!

**Pull request merged**